src/test/java/com/google/sps/servlets/BlobServletTest.java

```diff
@@ -32,8 +32,8 @@
 
   private final LocalServiceTestHelper helper =
       new LocalServiceTestHelper(new LocalUserServiceTestConfig(),
-                                 new LocalDatastoreServiceTestConfig(),
-                                 new LocalBlobstoreServiceTestConfig())
+                                 new LocalDatastoreServiceTestConfig()/*,
+                                 new LocalBlobstoreServiceTestConfig()*/)
```

Comment:
Is this supposed to be here?

Reply:
Yes, the blobstore POST testing is a WIP